include/use-case.h

```diff
@@ -342,6 +342,16 @@ int snd_use_case_get_list(snd_use_case_mgr_t *uc_mgr,
  *   - CaptureMasterType
  *      - type of the master volume control
  *      - Valid values: "soft" (software attenuation)
+ *   - CaptureChannelMap
+ *      - Remap channels using ALSA PCM channel mapping API notation
+ *        E.g. "2 3 0 1 -1 -1 -1 -1 -1 -1 -1" means, FL takes channel 2,
+ *        FR takes channel 3, RL takes channel 0, RL takes channel 1.
```

Comment:
We have already snd_pcm_chmap_parse_string() function, so we should accept probably this format.

But... Those parameters should be exposed via the ALSA's PCM API, otherwise all applications must parse and handle those values independently. I mean that it would be probably more appropriate to open just another PCM by the name which will be restricted to the specific configuration. It does not apply only to channel map, but the channel count, buffering etc. I will discuss this on the alsa mailing list further.

Reply:
Ok I haven't seen anything on the mailing list yet, I will keep my eyes peeled